test/e2e/create-branch.e2e.ts

```diff
@@ -9,25 +9,31 @@ test.describe("Create branch", () => {
 	test.beforeEach(async ({ page: newPage }) => {
 		page = newPage;
 		await jiraLogin(page, "admin");
+		await jiraAddProject(page);
 	});
 
 	test.use({
 		storageState: testData.jira.roles.admin.state
 	});
 
 	test.describe("cloud", () => {
+		// Create a fresh issue per test
 		test.beforeEach(async() => {
-			await jiraAppInstall(page);
+			await jiraCreateIssue(page);
 		});
+
 		test("When there are no GitHub connections", async () => {
-			await jiraAddProject(page);
-			await jiraCreateIssue(page);
 			await page.goto(data.urls.testProjectIssue);
 			await (page.locator("a[data-testid='development-summary-common.ui.summary-item.link-formatted-button']")).click();
 			const poppedUpPage = await page.waitForEvent("popup");
 			await poppedUpPage.waitForLoadState();
 			expect(poppedUpPage.getByText("Almost there!")).toBeTruthy();
 		});
 	});
+	// Clean up projects to avoid conflicts in future
+	// test.afterEach(async() => {
```

Comment:
couldn't quite get the project delete to work but left it there as Ill have another go at it!